### PR TITLE
Logger params

### PR DESCRIPTION
**Pull request by cliveseldon**
**What this PR does / why we need it**:

 * Adds helm parameters for new request logger settings
 * Add annotations for per SeldonDeployment override

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
```

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **cliveseldon**
You can assign the PR to them by writing `/assign @cliveseldon` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: doc/source/reference/upgrading.md [hunk 1]
```diff
@@ -19,6 +19,13 @@ Seldon Core adds support for Kubernetes 1.22 by upgrading all ValidatingWebhookC
   * Access required to modify files in the local folder are required so the application folder should be writable
   * The default base image now changes the owner of the /microservice folder to user 8888
 
+### Updated executor request logger settings
```
Comment (axsaucedo) [inline doc/source/reference/upgrading.md:22]: Nice - we should make sure this is also covered in the UPGRADING docs in the deploy side

File: executor/cmd/executor/main.go [hunk 2]
```diff
@@ -71,8 +71,8 @@ var (
 	filename          = flag.String("file", "", "Load graph from file")
 	hostname          = flag.String("hostname", "", "The hostname of the running server")
 	logWorkers        = flag.Int("logger_workers", 10, "Number of workers handling payload logging")
-	logWorkBufferSize = flag.Int("log_work_buffer_size", 10000, "Limit of buffered logs in memory while waiting for downstream request ingestion")
-	logWriteTimeoutMs = flag.Int("log_write_timeout_ms", 2000, "Timeout before giving up writing log if buffer is full. If <= 0 will immediately drop log on full log buffer.")
+	logWorkBufferSize = flag.Int("log_work_buffer_size", loghandler.DefaultWorkQueueSize, "Limit of buffered logs in memory while waiting for downstream request ingestion")
```
Comment (axsaucedo) [inline executor/cmd/executor/main.go:74]: Do these have to be params? Wondering whether env vars could be enough

File: helm-charts/seldon-core-operator/values.yaml [hunk 3]
```diff
@@ -66,6 +66,8 @@ executor:
 # For more information see the Production Integration for Payload Request Logging with ELK in the docs
   requestLogger:
     defaultEndpoint: 'http://default-broker'
+    workQueueSize: 10000
```
Comment (axsaucedo) [inline helm-charts/seldon-core-operator/values.yaml:69]: May need to also re-run the `make -C helm-charts/ docs`

File: executor/cmd/executor/main.go (hunk 2 above)
Comment (cliveseldon) [reply executor/cmd/executor/main.go:74]: As we have annotations at present do we need to add this as well?

**Comment by axsaucedo** [comment]
Nice one
/approve

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[axsaucedo](https://github.com/SeldonIO/seldon-core/pull/3738#issuecomment-970318311 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [axsaucedo]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**